.meta/sources/docker.toml

```diff
@@ -0,0 +1,33 @@
+[sources.docker]
+beta = true
+delivery_guarantee = "best_effort"
+guides = []
+output_types = ["log"]
+resources = []
+through_description = "the docker engine daemon"
+
+[sources.docker.options.include_containers]
+type = "[string]"
+examples = [
+    "<container-id>",
```

Comment:
Can we provide an actual example here, or remove this? The one below is a valid example.

Reply:
@binarylogic so I was unsure since it usually is a long random string of alphanumeric characters, but technically the docs explain what it is so I think its fine to not include this 👍